app/build.gradle

```diff
@@ -86,7 +86,11 @@ android {
     }
     packagingOptions {
         jniLibs {
-            pickFirsts += ['lib/armeabi-v7a/libtmpfile.so', 'lib/arm64-v8a/libtmpfile.so', 'lib/x86/libtmpfile.so', 'lib/x86_64/libtmpfile.so', 'lib/armeabi-v7a/libc++_shared.so', 'lib/arm64-v8a/libc++_shared.so', 'lib/x86/libc++_shared.so', 'lib/x86_64/libc++_shared.so']
+            // No need to pickFirst libc++_shared.so if all files are identical.
+            // They will be identical if NDK major version matches.
+            // NDK runtime problems may occur if NDK version mismatches,
+            // so comment this out, to get a compile error instead
+            // pickFirsts += ['**/libc++_shared.so']
```

Comment:
if I understand correctly, it sounds like we really want to comment this line out, do we? why would I prefer a runtime error over a compile error here?

Reply:
Some previous versions of Android Gradle Plugin would error out at compile time even if those files were identical, that's why we had that line there originally.

Only scenario for that is when we want to have mismatching NDK between libraries. That is how it currently is (before this PR):
OpenDocument.droid is compiled with NDK 25
pdf2htmlEX-Android-0.18.19 was compiled with NDK 23
wvWare-Android-1.2.8 was compiled with NDK 23

I'm not really sure if the runtime error is actually triggered. It all depends on what was changed in NDK's C++ standard library between those versions and it also depends on how we use those changed C++ standard library parts. It's in the NDK user guide though and not following that is just asking for trouble.

This whole thing is the reason why I'm publishing ndkports with multiple NDK versions (25 and 26), I just haven't implemented that part in pdf2htmlEX-Android and wvWare-Android. It was easy to do on ndkports, by just publishing the same library multiple times and listing the NDK version in the maven package name:
```
com.viliussutkus89.ndk.thirdparty:poppler-ndk26-static:24.01.0-beta-1
com.viliussutkus89.ndk.thirdparty:poppler-ndk25-static:24.01.0-beta-1
...
```

Ideally, I should put all those different versions in the same maven package and let gradle (prefab) sort it out. 

Haven't done any of the multiple NDK part on pdf2htmlEX-Android or wvWare-Android, because a hacky solution with multiple pdf2htmlEX-Android packages (like Poppler) would be a bit too confusing